front-end/README.md

```diff
@@ -276,7 +276,7 @@ Then add the block below to your `launch.json` file and put it inside the `.vsco
     "name": "Chrome",
     "type": "chrome",
     "request": "launch",
-    "url": "http://localhost:3000",
+    "url": "142.93.28.110:3005",
```

Comment:
not sure if this was a mistake or a legit change?

Reply:
Yea this was definitely a test I was running a while ago. I'll get that fixed.

Reply:
Awesome.